fusor_ovirt.spec

```diff
@@ -0,0 +1,37 @@
+Name:           fusor_ovirt
+Version:        0.1
+Release:        1%{?dist}
+Summary:        Scripts to interact with oVirt API
+
+Group:          Applications/System
+License:        GPLV3+ and ASL 2.0
```

Comment:
Basing this license on what we've specified for fusor-installer and katello-installer.